apps/api/src/beneficiary/beneficiary.controller.ts

```diff
@@ -61,7 +60,7 @@ export class BeneficiaryController {
     roles: [RealmViewSupporters.role, ViewSupporters.role],
     mode: RoleMatchingMode.ANY,
   })
-  async deleteMany(@Body() data: DeleteManyBeneficiaryDto) {
```

Comment:
@PepiPetrov Why do you change the DTO with plain string array? By doing that we can't validate the input anymore

Reply:
Maybe this PR should have included only one commit  https://github.com/podkrepi-bg/api/pull/126/commits/7cfa04acd7f5d80c0f1067b506ca981fb0b4ab4c instead of two commits